src/static/js/pluginfw/tsort.js

```diff
@@ -6,6 +6,7 @@
  *
  * @returns Array : topological sorted list of IDs
  **/
+'use strict';
```

Comment:
minor thing but I have been putting this as line 1.  I'm not sure if it's something we lint for tho as long as it's there and before any logic :)

Reply:
Not necessary, but I pulled it up.